python/llm/src/bigdl/llm/transformers/models/utils.py

```diff
@@ -202,6 +204,8 @@ def is_enough_kv_cache_room_4_36(past_key_value, idx, seq_len=1):
 
 def is_enough_kv_cache_room_4_31(past_key_value, seq_len=1):
     # to determinate if is enough kv cache room in transformers between 4.31 and 4.35
+    # seq_len for current seq len
+    # For llama like kv cache, i.e., [bs, n_head, seq_len, head_dim]
     return past_key_value is not None and \
         past_key_value[0].stride()[1] > \
```

Comment:
may change to `stride(1) >= (size(2) + seq_len) * size(3)`

Reply:
Yes. That's more readable.